src/utils/logger.js

```diff
@@ -0,0 +1,26 @@
+const {
+  LOG_LEVEL = 'all',
+} = process.env
+
+
+const levels = {
+  error: 1,
+  info: 2,
+  debug: 3,
+  all: 4,
+}
+
+const makeLogger = (name) => {
+  const defaultLog = { name }
+
+  const log = (level) => (data) => console.log(JSON.stringify({ ...defaultLog, level, ...data }))
+  const nop = () => undefined
+
+  return {
+    error: levels[LOG_LEVEL] >= 1 ? log('error') : nop,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
    error: levels[LOG_LEVEL] >= 1 ? log("error") : nop,
```